### DISCUSSION TITLE
How to integrate webcam video into animation?

### DISCUSSION THREAD

**Discussion by yatescr** [3b1b/manim#2249] [Q&A]
How do you integrate webcam video, or more simply, a pre-recorded .mkv file, with a scene, so that you can place the .mkv recording in a small rectangle (say, the lower left corner) of a generated scene, with audio and video commenting on the scene?

**Comment by yatescr** [comment]
Here is an example of my question: https://howthefouriertransformworks.com/courses/?utm_source=LinkedIn&utm_medium=Post&utm_campaign=Free+Trial

**Reply by aguemoug** [reply]
i guess its simple video editing
